docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -131,38 +126,32 @@ done
 . Get the count of not-OK source databases:
```

Comment:
If "not-OK" is not code, possibly rephrase to "Get the count of source databases that do not have an OK status:"

Reply:
Just see `NOK` in the code snippets further down now so maybe revise my last comment to something like "Get the count of source databases with the `NOK` (not-OK), status:"

if that makes sense

Reply:
@bogdando Can you confirm that I can rewrite this step to say the following?
"Get the count of source databases with the NOK (not-OK) status:"